anybox/recipe/odoo/base.py

```diff
@@ -138,6 +140,21 @@ class BaseRecipe(object):
     directory right next to the ``openerp`` package.
     """
 
+    with_odoo_requirements_file = False
+    """Whether attempt to use the 'requirements.txt' shipping with Odoo"""
```

Comment:
On 07/14/2015 03:55 PM, Vincent Vinet wrote:
>
> In anybox/recipe/odoo/base.py
> <https://github.com/anybox/anybox.recipe.odoo/pull/45#discussion_r34569819>:
>
> > @@ -138,6 +140,21 @@ class BaseRecipe(object):
> >      directory right next to the ``openerp`` package.
> >      """
> >  
> > +    with_odoo_requirements_file = False
> > +    """Whether attempt to use the 'requirements.txt' shipping with Odoo"""
>
> Nitpick: strings are not comments, even though this is done in a few
> places...
>
Actually, that'd be a docstring, not a comment, but you're right in that
docstrings for regular attributes aren't part of the language.
However, this way of doing is popular enough to have Sphinx support, and
that's what I'm after.

See the result here:
http://docs.anybox.fr/anybox.recipe.odoo/current/apidoc/anybox.recipe.odoo.html#anybox.recipe.odoo.base.BaseRecipe.is_git_layout

-- 
Georges Racinet
Anybox SAS, http://anybox.fr
Téléphone: +33 6 51 32 07 27
GPG: 0x33AB0A35, sur serveurs publics